src/wp-includes/blocks.php

```diff
@@ -881,6 +881,7 @@ function excerpt_remove_blocks( $content ) {
 		'core/heading',
 		'core/html',
 		'core/list',
+		'core/list-item',
```

Comment:
Should we move `core/list` to `allowed_wrapper_blocks`? 🤔 Also I guess we have the same issue for `core/quote` as it changed to support `innerBlocks` and we should move this to `allowed_wrapper_blocks` as well.

Reply:
I have thought about this issue again and it may be more complicated than I imagined.

If allowed by `allowed_wrapper_blocks`, the wrapper block will be removed and only the `innerBlocks` will be rendered. That is, the `ul/ol` element is removed and only the `li` elements are rendered.

Rendering only inner blocks is the expected behavior for `columns/column/group` block. However, for the list block only, the wrapper block must also be rendered.

So I left the allowed list as is and added here a special condition for the list block. There may be other more ideal approaches.